sdk/vision/azure-ai-vision-imageanalysis/azure/ai/vision/imageanalysis/_operations/_patch.py

```diff
@@ -0,0 +1,91 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+from typing import Any, Optional, List, Union
+from .. import models as _models
+from ._operations import ImageAnalysisClientOperationsMixin as ImageAnalysisClientOperationsMixinGenerated
+
+class ImageAnalysisClientOperationsMixin(ImageAnalysisClientOperationsMixinGenerated):
+
+    def analyze(
+        self,
+        image_content: Union[str, bytes],
```

Comment:
Should IO[bytes] be included here?

Reply:
Confirmed offline with the service team that streaming (HTTP chunking) is not supported by the Image Analysis service. Following discussion with Krista, resolving this comment.